R/render-table.R

```diff
@@ -3,33 +3,103 @@
 #' Creates a reactive table that is suitable for assigning to an \code{output}
 #' slot.
 #'
-#' The corresponding HTML output tag should be \code{div} and have the CSS class
-#' name \code{shiny-html-output}.
+#' The corresponding HTML output tag should be \code{div} and have the CSS
+#' class name \code{shiny-html-output}.
 #'
 #' @param expr An expression that returns an R object that can be used with
 #'   \code{\link[xtable]{xtable}}.
-#' @param ... Arguments to be passed through to \code{\link[xtable]{xtable}} and
-#'   \code{\link[xtable]{print.xtable}}.
+#' @param striped,condensed,hover,bordered Logicals: if \code{TRUE}, apply
+#'   the corresponding Bootstrap table format to the output table.
```

Comment:
Sure, that would be possible too. I guess this way felt more natural to me, but I agree that 4 args instead of 1 may be a little overwhelming to the user. Now, I'm quite torn between the two approaches... Winston, thoughts?